### PR TITLE
sports betting example 

### PR DESCRIPTION
**Pull request by GreyMcCarthy**
Added a sports betting example similar to the election example.

### CODE REVIEW

**Comment by GreyMcCarthy** [comment]
Added the `wallet-sports-betting-exercise`  and `oracle-sports-betting-exercise` to `sidebars.json`

File: docs/oracle/oracle-sports-betting-example.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+You will also need a the `bitcoin-s-cli` command line tool to interact with the server.
+You can find how to build this [here](../applications/cli.md)
+
+After building the oracle server, you will need to start it with
+
+```
+./app/oracle-server/target/universal/stage/bin/bitcoin-s-oracle-server
+```
+
+## Rocky V.S. Drago 2022 Boxing Championship
```
Comment (Christewart) [inline docs/oracle/oracle-sports-betting-example.md:19]: Great example! :joy:

**Review by Christewart — APPROVED** [review]
Awesome job with this!

**Comment by Christewart** [comment]
fixes #4371

**Pull request merged**